monitoring/values/kibana.values.yaml

```
@@ -14,7 +14,7 @@ files:
     server.name: kibana
     server.host: "0"
     server.basePath: "/kibana"
-    elasticsearch.url: http://elascticsearch-elasticsearch-coordinating-only:9200
+    elasticsearch.url: http://elascticsearch-elasticsearch-client.elasticsearch:9200
```

Typo
```suggestion
    elasticsearch.url: http://elasticsearch-elasticsearch-client.elasticsearch:9200
```

Also can we just not have `http://elasticsearch-client.elasticsearch:9200`?I think there's enough occurrences of the word `elasticsearch`